piicatcher/command_line.py

```diff
@@ -88,40 +89,40 @@ def version_callback(value: bool):
 # pylint: disable=too-many-arguments
 @app.callback(invoke_without_command=True)
 def cli(
-    log_level: str = typer.Option("WARNING", help="Logging Level"),
-    log_data: bool = typer.Option(False, help="Choose output format type"),
-    log_scan: bool = typer.Option(False, help="Log data that was scanned"),
-    config_path: Path = typer.Option(
-        typer.get_app_dir("tokern"), help="Path to config directory"
-    ),
-    output_format: OutputFormat = typer.Option(
-        OutputFormat.tabular, case_sensitive=False
-    ),
-    catalog_path: Path = typer.Option(
-        None, help="Path to store catalog state. Use if NOT using a database"
-    ),
-    catalog_host: str = typer.Option(
-        None, help="hostname of Postgres database. Use if catalog is a database."
-    ),
-    catalog_port: int = typer.Option(
-        None, help="port of Postgres database. Use if catalog is a database."
-    ),
-    catalog_user: str = typer.Option(
-        None, help="user of Postgres database. Use if catalog is a database."
-    ),
-    catalog_password: str = typer.Option(
-        None, help="password of Postgres database. Use if catalog is a database."
-    ),
-    catalog_database: str = typer.Option(
-        None, help="database of Postgres database. Use if catalog is a database."
-    ),
-    catalog_secret: str = typer.Option(
-        dbcat.settings.DEFAULT_CATALOG_SECRET,
-        help="Secret to encrypt sensitive data like passwords in the catalog.",
-    ),
-    version: Optional[bool] = typer.Option(
-        None, "--version", callback=version_callback, is_eager=True
-    ),
+        log_level: str = typer.Option("WARNING", help="Logging Level"),
```

Comment:
There are whitespace changes. Can you revert these changes?

Reply:
Sure.